wuvt/templates/base.html

```diff
@@ -35,24 +35,24 @@
     <div id="mainheader">
         <h1>
             <a href="{{ url_for('blog.index') }}" title="{{ config.STATION_NAME }}">
-                WUVT: Radio for Everyone! 90.7 FM
-            </a>
-        </h1>
+                <img id="logo" src="{{ url_for('static', filename='img/logo.svg') }}" alt="WUVT: Radio for Everyone! 90.7 FM" />
```

Comment:
Why are you making this an image? fundamentally it is a link, that just happens to look like an image.